src/client-private.ts

```diff
@@ -143,7 +232,47 @@ function onSession (this: PureCloudWebrtcSdk, session) {
       this.emit('sessionEnded', session, reason);
     });
     this.emit('sessionStarted', session);
-  });
+    // if authenitcation is jwt (meaning guest/annoyomous) and sdkType is screenshare
+    // then we are sharing our screen
+  } else if (this._guest && this._sdkType === 'screenshare' && this._jwt) {
+    log.call(this, 'debug', 'sdkType is \'screenshare\' and there is a jwt');
+    if (temporaryOutboundStream) {
+      log.call(this, 'debug', 'temporaryOutboundStream exists. Adding stream to the session and setting it to _outboundStream');
+      session.addStream(temporaryOutboundStream);
+      session._outboundStream = temporaryOutboundStream;
+      temporaryOutboundStream = null;
+    }
+
+    /* don't attach? or should we and then hide it?
```

Comment:
Yes and no. Guest screen shares won't have any streams in session.streams, so there's no need to attach any incoming media.